issuetracker/github/github_test.go

```diff
@@ -0,0 +1,31 @@
+package github
+
+import (
+	"fmt"
+	"testing"
+)
+
+func Test_IssueTracker_issueAPIOrigin(t *testing.T) {
+	var tests = []struct {
+		input string
+		want  string
+	}{
+		{"https://github.com/preslavmihaylov/todocheck", "https://api.github.com/repos/preslavmihaylov/todocheck/issues/"},
+		{"github.com/preslavmihaylov/todocheck/", "https://api.github.com/repos/preslavmihaylov/todocheck/issues/"},
+	}
+
+	for _, tt := range tests {
+
+		var it IssueTracker
+
+		testname := fmt.Sprintf("%q", tt.input)
+		t.Run(testname, func(t *testing.T) {
+			it.Origin = tt.input
+			res := it.issueAPIOrigin()
```

Comment:
could you make the test against `IssueURLFor` instead of this private function? The private functions are implementation details which are unstable & can change anytime. The public API is supposed to be stable & shouldn't change very often

Reply:
done